lib/queue_kit/signal_checker.rb

```diff
@@ -36,7 +36,6 @@ def trap_signal(sig)
       debug :setup, sig
 
       old_handler = trap sig do
-        debug :trap, sig
```

Comment:
@kdaigle I'm not 100% sure I understand but as far as I can understand from the `debug` method below and following the chain this is also just an instrumentation call. Without it added I see the `ThreadErrors` at the later stage of this call.